src/modules/sub-items/sub.items.module.js

```diff
@@ -256,12 +266,44 @@ export default class SubItemsModule extends Component {
         this.setState(() => ({ activeView }));
     }
 
+    onItemSelect(contentId, isSelected) {
+        const { items } = this.state;
+        const itemIdx = items.findIndex((item) => item.content._id == contentId);
```

Comment:
What does `itemIdx` mean? Please, use meaningful variable names. The code should be readable like a text in book.